scripts/publish-ghpages.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+set -o errexit -o nounset
+
+# User info
+git config user.name "patternfly-build"
+git config user.email "patternfly-build@redhat.com"
+git config --global push.default simple
+
```

Comment:
Will need to check for patternfly/patternfly-org and the master branch. The travis.yml file does this via the deploy properties. Otherwise, any clone can push to GitHub and potentially corrupt the site.

Take a look at the publish.sh script for patternfly. It tests $TRAVIS_REPO_SLUG, $TRAVIS_TAG and $TRAVIS_BRANCH.